cfy_manager/components/manager/config/cloudify_manager.te

```diff
@@ -0,0 +1,27 @@
+module cloudify_manager 1.0;
+
+require {
+        type httpd_t;
+        type unreserved_port_t;
+        type ntop_port_t;
+        class tcp_socket name_connect;
+
+        type postgresql_port_t;
+        type haproxy_t;
+        type gatekeeper_port_t;
+        class tcp_socket { name_bind name_connect };
+}
+
+#============= httpd_t ==============
+
+# ntop_port_t for `node` on :3000
+allow httpd_t ntop_port_t:tcp_socket name_connect;
```

Comment:
Let's make a cloudify_port_t containing all the ports we care about and just use it. I'm not sure if that can be done here or has to be done using commands like 'semanage port -a -t ssh_port_t -p tcp 2222'